src/cljs/rems/application_list.cljs

```diff
@@ -58,6 +58,12 @@
                           {:class (when (application-util/form-fields-editable? app)
                                     "text-highlight")}
                           value]})
+           :todo (let [value (localize-todo (:application/todo app))]
+                   {:value value
+                    :td [:td.state
+                         {:class (when (str/includes? (str (:application/todo app)) "-your-")
```

Comment:
Hmm, it would probably be cleaner if there was an explicit list of todo types that are highlighted.